### PR TITLE
Fei and Anxipal fluffs

### PR DESCRIPTION
**Pull request by FlattestGuitar**
/obj/item/clothing/suit/jacket/fluff/jacksvest
```
Name: Jack's Vest
Desc: It's a vest; doesn't do anything but look cool.
Character: Jack Harper
Ckey: Anxipal
```

obj/item/device/fluff/fei_gasmask_kit
```
Name: Prescription Gas Mask
Desc: It's a gas mask conversion kit. Just looks cool.
Character: Tariq Yon-Dale
Ckey: Fei Hazelwood
```

### CODE REVIEW

**Comment by Spartan6** [comment]
The vest on top looks an awful lot like armour.

**Comment by taukausanake** [comment]
Tunnel Snakes rule. Also looks like one of those poofy jacket vests and I like it.

Can I pretend that the mask is actually the face of a Kidan that was turned into a mask?

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -742,6 +769,17 @@
 	species_fit = null
 	sprite_sheets = null
 
+/obj/item/clothing/suit/jacket/fluff/jacksvest // Anxipal: Jack Harper
+	name = "Jack's vest"
+	desc = "A rugged leather vest with a tag labelled \"President.\""
```
Comment (Citinited) [inline code/modules/customitems/item_defines.dm]: The full stop should be after the quotation mark, not before

**Comment by StreakyHaddock** [comment]
a nerd's mask for a giant nerd B)

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -731,7 +758,7 @@
 	sprite_sheets = null
 
 /obj/item/clothing/suit/jacket/fluff/kidosvest // Anxipal: Kido Qasteth
-	name = "Kido's Vest"
+	name = "Kido's vest"
```
Comment (tigercat2000) [inline code/modules/customitems/item_defines.dm]: Don't change other fluff items needlessly- The above is fine due to fixing a spelling error, but there's no reason to change this capitalization.

**Pull request merged**